databases/week2/part4.sql

```diff
@@ -0,0 +1,49 @@
+-- Creating a database
```

Comment:
Makes sense, maybe "Menu" should be called "Dishes", but that is a really minor thing. Also, order_id probably should not be in the UML diagram in the "Waiter" table since it's not used.